recipe-server/normandy/recipes/models.py

```diff
@@ -341,6 +346,18 @@ class Action(models.Model):
 
     arguments_schema_json = models.TextField(default='{}', validators=[validate_json])
 
+    errors = {
+        'duplicate_branch_slug': ('Feature branch slugs must be unique within an '
+                                  'experiment: "{slug}" is duplicated.'),
```

Comment:
Do we do hanging indents in Python? We may not have explicitly decided on this. I personally prefer to avoid them, and I think it's uncommon in our codebase.